Non-Interactive-Protocol.md

```diff
@@ -11,115 +11,111 @@
 cross-layer mapping. <sup>[Cross-layer Mapping](https://arxiv.org/pdf/2007.00764.pdf)</sup>
 
-We recommend to not produce local non-adaptor signatures for transaction until their broadcast
-is required to avoid their misuage in case of site compromise, assuming an external signer.
+We recommend clients should not produce local (non-adaptor) signatures for CETs until they are about to be broadcast
+in order to avoid their misuse in case of site compromise, assuming an external signer.
 
-Further, we recommend duplicate backups of counterparty signatures to avoid relying on
-counterparty honest behavior in case of primary storage failure.
+Furthermore, we recommend duplicated backups of counter-party signatures be stored to avoid relying on
+honest counter-party behavior in the case of primary storage failure.
 
-Transaction finalization is not implying a mandatory key material access.[ FIXME: Client Key Management. ]
+Transaction finalization does not imply a mandatory key material access. [ FIXME: Client Key Management. ]
 
 # Fee bumping policy
 
-An onchain state machine must take actions based on different timers parameterized by three
-values: `congestion_bump_frequency`/`security_bump_frequency`/`security_point`/`refund_security_point`.
-Their definition and setting recommendations are laid out in this section, their usage is precised
-in following sections.
+An on-chain state machine must take actions based on different clocks parameterized by three
+values: `congestion_bump_frequency`/`security_bump_frequency`/`funding_security_point`/`refund_security_point`.
+These values' definitions and setting recommendations are laid out in this section and their usage is made
+more precise in following sections.
 
-A `security_point` is a client heuristic after which the confirmation of the funding transaction
-is a matter of contract safety, otherwise the user is exposed to a bad-player double-spend if either
-the outcome odds are modified or a signature oracle is released. This point is function of event
-signing time, which may be static or dynamic, we defer point qualification to client.
+A `security_point` is a client-side heuristic time/block-stamp after which the confirmation of a corresponding
+transaction is a matter of contract safety.
+If confirmation does not occur soon after this point, the user can be vulnerable to a malicious double-spend if either
+the outcome odds change or an oracle signature is released.
+This point is determined as a function of event signing time, which may be static or dynamic.
+We defer point qualification to client.
 
-If the signing time is static (block height, epoch) or predictable we recommend setting the 
-`security_point` with a confirmation buffer of 20 blocks compared to signing time.
+If the signing time is static (block height, epoch) or predictable, we recommend setting 
+`security_point`s to have a confirmation buffer of 20 blocks from signing time.
```

Comment:
I'm not sure I understand how to use that in practice. Does it relate to the refund time?

Reply:
This is the point at which you start more aggressively upping the fee (and do so every block)

Reply:
Maybe it's just me but I am a bit confused. IIUC there are two `security_point`s. So does this sentence apply too both of them? And so `refund_security_point` should be 20 blocks after expected signing time? I don't think that is what is meant and sorry if I'm just slow to get it but it really doesn't read well to me.
(I need to find time to just make a state machine drawing for this section it would be 100x clearer IIMHO)

Reply:
Thanks now I get it 🙇